.github/CODEOWNERS

```diff
@@ -0,0 +1,16 @@
+# Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
+# See https://help.github.com/articles/about-code-owners/
+
+/eng/SourceBuild*                                                                 @dotnet/source-build
```

Comment:
This is a strange error, @dotnet/source-build does exist:

> Unknown owner on line 4: make sure the team @dotnet/source-build exists, is publicly visible, and has write access to the repository

Reply:
I believe that the team has to have write access.